proposed/extended-coding-style-guide.md

```diff
@@ -539,13 +539,35 @@ namespace Vendor\Package;
 
 class ReturnTypeVariations
 {
-    public function functionName(?string $arg1, ?int $arg2): ?string
+    public function functionName(?string $arg1, ?int &$arg2): ?string
     {
         return 'foo';
     }
 }
 ~~~
 
+When using the reference operator `&` before an argument, there MUST NOT be
+a space after it, like in the previous example.
+
+Variadic argument three dots MUST NOT have space before argument name:
```

Comment:
the wording of this sentence seems odd, but I'm not a native speaker, so I'm not sure how to write it properly

Reply:
@michaelcullum would you please take a look?

Reply:
"There MUST NOT be a space between the variadic three dots and the argument name."  ?